### DISCUSSION TITLE
14.1

### DISCUSSION THREAD

**Discussion by HiMoTeP12** [cathery/sys-con#290] [Troubleshooting] [answered]
Hello, FW 14.1 break sys-con (ps4 controler, not tested on xbox) can you do something plz ?

AMS : 1.3.1
FW : 14.1

Thank's

**Comment by eadmaster** [comment]
Just tested the latest release on 15.01 and it is working fine.
(Some 3rd party controllers are not recognized.)

**Comment by HiMoTeP12** [comment, marked as answer]
Thx, i give a try for [Mission Control](https://github.com/ndeadly/MissionControl) worked for me
